scripts/template_upload_vsphere.py

```diff
@@ -306,9 +306,28 @@ def make_kwargs_vsphere(cfmeqe_data, provider):
     return kwargs
 
 
-def upload_template(client, hostname, username, password,
-                    provider, url, name, provider_data):
+def cleanup_old_templates(provider, client, delete_templates):
```

Comment:
This function is almost identical in each case, let's put this somewhere common to all of them so that if/when we want to change it, we don't have to change all 4 files.